modules/hybrid-app/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@expensify/react-native-hybrid-app",
+  "version": "0.0.0",
```

Comment:
Definitely NAB, but maybe we could increment the version 😄

Reply:
I'd keep it that way because we have `0.0.0` in background-task too